test/api-digester/import-module-with-package-name.swift

```diff
@@ -0,0 +1,56 @@
+// REQUIRES: VENDOR=apple
+
+// RUN: %empty-directory(%t.mod)
+// RUN: %empty-directory(%t.sdk)
+// RUN: %empty-directory(%t.module-cache)
+// RUN: split-file %s %t
```

Comment:
I'd add a `// RUN: %empty-directory(%t)` if we generate Swift files there to make sure we don't pick up stale ones.